From 097acab8d937ac86018db1339e73745fd0c0f91d Mon Sep 17 00:00:00 2001
From: Adam Ford <aford@beaconembedded.com>
Date: Wed, 9 Jul 2025 09:16:50 -0500
Subject: [PATCH] test-bat:  Fix A53 core name on 8MM, 8MN, and 8MP

The proper name of the clock used on the A53 is _core
and not _div.

Signed-off-by: Adam Ford <aford@beaconembedded.com>

diff --git a/test/bat/bat-cpufreq-freq-check.sh b/test/bat/bat-cpufreq-freq-check.sh
index 0c41456..89a9965 100755
--- a/test/bat/bat-cpufreq-freq-check.sh
+++ b/test/bat/bat-cpufreq-freq-check.sh
@@ -25,8 +25,8 @@ get_arm_clk_name()
         if [ "$cpu_part" = "0xd08" ]; then
             echo "a72_div";
         elif [ "$cpu_part" = "0xd03" ]; then
-            if [ -d /sys/kernel/debug/clk/arm_a53_div ]; then
-                echo "arm_a53_div"
+            if [ -d /sys/kernel/debug/clk/arm_a53_core ]; then
+                echo "arm_a53_core"
             else
                 echo "a53_div"
             fi
-- 
2.43.0

